### PR TITLE
chore: added missing ids

### PR DESCRIPTION
**Pull request by harish-sethuraman**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/HEAD/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "X" next to an item)

[ ] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-change-proposal.md))
[ ] Add autofix to a rule
[ ] Add a CLI option
[ ] Add something to the core
[ ] Other, please explain:

#### What changes did you make? (Give an overview)
- To prefill description with the configuration of playground we need to add these ids.

#### Is there anything you'd like reviewers to focus on?
- Shall I add ids to fields that we dont use as well? so that we dont have to add it in future?

### CODE REVIEW

**Comment by nzakas** [comment]
I'd say let's just go ahead and add IDs to all of the fields so we don't have to revisit later.

**Comment by harish-sethuraman** [comment]
Added for all the entries in `bug-report.yml`

**Comment by amareshsm** [comment]
> Added for all the entries in `bug-report.yml`

Could you confirm if you've pushed your changes?

**Review by amareshsm — APPROVED** [review]
LGTM. Waiting for the 2nd approval.

**Comment by nzakas** [comment]
@amareshsm don't forget to move the PR to "Second Review Needed" in the Triage board when you're waiting on a second approval.

**Pull request merged**
